doc/api/cli.md

```diff
@@ -595,6 +595,29 @@ Path to the file used to store the persistent REPL history. The default path is
 `~/.node_repl_history`, which is overridden by this variable. Setting the value
 to an empty string (`''` or `' '`) disables persistent REPL history.
 
+### `NODE_UNHANDLED_REJECTION=state`
+<!-- YAML
+added: REPLACEME
+-->
+
+Setting this environment variable allows fine grained control over the behavior
+of unhandled rejections. This may be set to either one of `SILENT`, `WARN`,
+`ERROR_ON_GC` or `ERROR` while the default behavior without using the
+environment variable is the same as `WARN`. `WARN` reflects the
+[`unhandledRejection hook`][].
+
+Setting the environment variable to `SILENT` prevents unhandled rejection
+warnings from being logged, even if no [`unhandledRejection hook`][] is set.
+
+If set to `ERROR_ON_GC` unhandled rejections that are garbage collected are
+turned into exceptions.
+
+If set to `ERROR` any unhandled rejection is going turn the rejection into an
```

Comment:
is going to

Reply:
turn the _unhandled rejection_ into an _uncaught_ exception.